src/Okta.Sdk.IntegrationTests/EventHooksClientShould.cs

```diff
@@ -0,0 +1,382 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Xunit;
+using Xunit.Sdk;
+
+namespace Okta.Sdk.IntegrationTests
+{
+    public class EventHooksClientShould
+    {
+        private const string SdkPrefix = "dotnet_sdk";
+        private const string EventType = "EVENT_TYPE";
+
+        private static readonly string[] TestEventItems = new string[]
+        {
+            "user.lifecycle.create",
+            "user.lifecycle.activate",
+        };
+
+        private static readonly string[] UpdatedTestEventItems = new string[]
+        {
+            "user.lifecycle.create",
+            "user.lifecycle.activate",
+            "user.lifecycle.deactivate",
+        };
+
+        private static readonly EventHookChannel TestEventHookChannel = new EventHookChannel
+        {
+            Type = "HTTP",
+            Version = "1.0.0",
+            Config = new EventHookChannelConfig
+            {
+                Uri = "https://www.example.com/eventHooks",
+                Headers = new List<IEventHookChannelConfigHeader>
+                {
+                    new EventHookChannelConfigHeader
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value",
+                    },
+                },
+                AuthScheme = new EventHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key",
+                },
+            },
+        };
+
+        private static readonly EventHookChannel UpdatedTestEventHookChannel = new EventHookChannel
+        {
+            Type = "HTTP",
+            Version = "1.0.0",
+            Config = new EventHookChannelConfig
+            {
+                Uri = "https://www.example.com/eventHooksUpdated",
+                Headers = new List<IEventHookChannelConfigHeader>
+                {
+                    new EventHookChannelConfigHeader
+                    {
+                        Key = "X-Test-Header",
+                        Value = "Test header value updated",
+                    },
+                },
+                AuthScheme = new EventHookChannelConfigAuthScheme
+                {
+                    Type = "HEADER",
+                    Key = "Authorization",
+                    Value = "Test-Api-Key-Updated",
+                },
+            },
+        };
+
+        public EventHooksClientShould()
+        {
+            DeleteAllEventHooks().Wait();
+        }
+
+        [Fact]
+        public async Task CreateEventHook()
+        {
+            var testClient = TestClient.Create();
+            var testEventHookName = $"{SdkPrefix}:{nameof(CreateEventHook)} Test Event Hook Name";
+
+            var createdEventHook = await testClient.EventHooks.CreateEventHookAsync(new EventHook
```

Comment:
Nit: Move `new EventHook` to the next line. [This rule](https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1116.md) should be covered by Stylecop. Does the warning show up on VS?

Reply:
Visual Studio isn't giving me those warnings.  Do we have a stylecop settings file I can import?

Reply:
I can see that the IT project has StyleCop dependencies and also a stylecop.json file. Warnings should show up on Visual Studio 🤔

Reply:
I'll fix the code and review my stylecop settings separately.